web/packages/core/src/internal/player/inner.tsx

```diff
@@ -2328,3 +2327,31 @@ function parseAllowScriptAccess(
             return null;
     }
 }
+
+/**
+ * Detect direction (LTR/RTL) of the preferred language of the browser.
+ *
+ * @returns 'ltr' or 'rtl' depending on the direction, `null` when unable to detect
```

Comment:
Would it make sense to just return 'ltr' if unable to detect, if that's the default anyway, so we can just do:

`this.contextMenuElement.dir = detectBrowserDirection()`

Reply:
Done!